### ISSUE TITLE
maize tassel branch PO terms  - rename long, short to second, third order

### ISSUE THREAD

**Issue by mlschaeffer** [Planteome/plant-ontology#660]
A suggestion - see last sentence
Looking to select PO terms to annotate maize trait 'tassel angle', defined as "average angle of branches, estimated in the field; 0 degrees, side branches are perpendicular to the central spike; 90, side branches are parallel to the central spike. From Upadyayula et al 2006".

These seem to be secondary order inflorescence branch, PO:0006322. Since maize flower parts usually require adding a second PO term to make retrieval work, [or is this now fixed?]; it seems I would then use this term:
PO:0020126  tassel inflorescence or PO:0009101 long lateral tassel branch. aka second order tassel branch

Curiously, there is a term "PO:0009102 short tassel branch" with a definition "...may arise from either or both the long lateral branch and the central spike of tassel in maize. ", but is aka "third order tassel branch". 

To keep in parallel with the main PO, vs the tassel sidekick terms, suggest renaming short and long tassel branches, since the length seems not to be the issue, but rather the order (second, third).

**Comment by DarioJCavaliere** [comment]
I agree, position is much more informative from a developmental and anatomical perspective.
